tests/integration/views/account/test_share_surveys.py

```diff
@@ -8,7 +8,7 @@
     url_get_respondent_party, url_get_survey, business_party, survey
 
 url_get_business_details = f"{app.config['PARTY_URL']}/party-api/v1/businesses"
-url_get_user_count = f"{app.config['PARTY_URL']}/party-api/v1/share-survey-users-count"
+url_get_user_count = f"{app.config['PARTY_URL']}/party-api/v1/share-survey-users-count?business_id={business_party['id']}&survey_id={'02b9c366-7397-42f7-942a-76dc5876d86d'}"
```

Comment:
The survey id doesn't need to be in curly braces.  You can do `survey_id=02b9c366-7397-42f7-942a-76dc5876d86d'`

Reply:
its just the test case will fix it with my next pr